python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -669,6 +689,23 @@ def _get_cls_name(serialized: Optional[Mapping[str, Any]]) -> str:
     return ""
 
 
+KeyType = TypeVar("KeyType")
+ValueType = TypeVar("ValueType")
+
+
+def _get_first_value(
+    mapping: Mapping[KeyType, ValueType], keys: Iterable[KeyType]
+) -> Optional[ValueType]:
+    """
+    Returns the value corresponding to the first matching key, or None if no key
+    is present.
+    """
+    return next(
+        (mapping[key] for key in keys if key in mapping),
```

Comment:
in case they're both present
```suggestion
        (item for key in keys if (item := mapping.get(key)) is not None),
```